.jenkins/pytorch/common.sh

```diff
@@ -181,11 +181,12 @@ function get_exit_code() {
 }
 
 function file_diff_from_base() {
+  BASE_REF=${BASE_REF:-master}
```

Comment:
why would we not have a BASE_REF?  I don't see any reason why we want to encode master as magic here instead of always setting the BASE_REF?

Reply:
I usually just put these here as a marker that it is an environment variable that gets populated somewhere and that a sane value of it would be `master`.

In any case there's no real reason why we wouldn't have base ref, but I like having a nice default just in case.